packages/components/socialFeed/Map/Map.web.tsx

```diff
@@ -103,6 +110,43 @@ const MapManager = ({
   return null;
 };
 
+const DynamicMinZoom = ({
+  setMinZoom,
+}: {
+  setMinZoom: Dispatch<SetStateAction<number>>;
+}) => {
+  const map = useMap();
+
+  useEffect(() => {
+    const lat = 45;
+    const lng = 90;
+
+    const adjustedWorldMapBounds: LatLngBoundsLiteral = [
+      [-lat, -lng],
+      [lat, lng],
+    ];
+
+    const updateMinZoom = () => {
```

Comment:
We should not defined the function inside of useEffect which makes the code less readable.

I suggest sth like
```
useEffect(() => {
    !!map && updateMinZoom(map);
    .......
}, [[map, setMinZoom])
```

Reply:
I disagree, we followed this in berty and it made the code very hard to read IMO

Reply:
you can extract a hook though if you really want

```
const useUpdateMinZoom = (map) => {
    useEffect(...)
}
```

Reply:
fixed here: https://github.com/TERITORI/teritori-dapp/pull/1399/commits/066f8ccb7aa33572b8b03cb7b1b730e55c4b0ea6